companyweb_base/tests/test_api_cweb.py

```diff
@@ -41,7 +54,7 @@ def _get_vcr_kwargs(self, **kwargs):
 
     @freeze_time("2021-03-23")  # because the login hash includes the date
     def test_cweb_button(self):
-        self.p1.with_context(lang="fr_Fr").cweb_button_enhance()
+        self.p1.with_context(lang="fr_FR").cweb_button_enhance()
```

Comment:
The French translators community wants to remove `fr_FR` and use `fr` only..
@xavier-bouquiaux do you remember why we use fr_FR here?